extension/src/popup/components/WarningMessages/index.tsx

```diff
@@ -446,7 +445,7 @@ export const ScamAssetWarning = ({
               )}
             </div>
             <div className="ScamAssetWarning__btns">
-              {!isValidatingSafeAssetsEnabled && !isSendWarning && (
+              {!isSendWarning && (
```

Comment:
sorry for another slightly unrelated change - but product says that a blockaid warning should not block the ability to sign something